### PR TITLE
chore: Remove unneeded success messages

### LINKED ISSUE #1843: Remove success messages
Almost all success messages are intrusive
Events loaded successfully, profile loaded successfully, notifications loaded successfully.

Remove all success messages accept on actions like check in/out, deletion and modification.

Only show messages on errors and the above actions

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1843 

Changes: 

Removed unneeded success messages like `Organizer Details Loaded Successfully`, `Notifications Loaded Successfully` etc.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Refresh Complete not removed

**Comment by ShridharGoel** [comment]
Updated.

File: app/src/main/java/com/eventyay/organizer/core/attendee/history/CheckInHistoryView.java [hunk 1]
```diff
@@ -6,5 +6,5 @@
 import com.eventyay.organizer.common.mvp.view.Refreshable;
 import com.eventyay.organizer.data.attendee.CheckInDetail;
 
-public interface CheckInHistoryView extends Progressive, Erroneous, Refreshable, Emptiable<CheckInDetail> {
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/attendee/history/CheckInHistoryView.java]: Don't remove this. Just leave the onRefreshComplete method empty with a comment

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/attendee/history/CheckInHistoryView.java]: Okay

**Comment by ShridharGoel** [comment]
Updated.

File: app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailViewModel.java [hunk 2]
```diff
@@ -109,7 +108,6 @@ public void uploadImage(ImageData imageData) {
                 .doOnSubscribe(disposable -> progress.setValue(true))
                 .doFinally(() -> progress.setValue(false))
                 .subscribe(uploadedImage -> {
-                    success.setValue("Image Uploaded Successfully");
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailViewModel.java]: This should remain as it is a response to user action

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/organizer/detail/OrganizerDetailViewModel.java]: Done.

**Pull request merged**